node-runner-cli/api/CoreApiHelper.py

```diff
@@ -60,14 +63,38 @@ def network_status(self, print_response=False):
             except ApiException as e:
                 Helpers.handleApiException(e)
     
-    def engine_configuration(self):
+    def engine_configuration(self, print_response=False):
         with core_api.ApiClient(self.system_config) as api_client:
             api_client = self.set_basic_auth(api_client, "admin", "admin")
             try:
                 api = engine_api.EngineApi(api_client)
                 response: EngineConfigurationResponse = api.engine_configuration_post(
                     EngineConfigurationRequest(self.network_configuration().network_identifier))
-                return response
+                return self.handle_response(response, print_response)
+            except ApiException as e:
+                Helpers.handleApiException(e)
+    
+    def vote(self, print_response=False):
+        with core_api.ApiClient(self.system_config) as api_client:
+            api_client = self.set_basic_auth(api_client, "superadmin", "superadmin")
+            try:
+                api = key_api.KeyApi(api_client)
+                request = UpdateVoteRequest(network_identifier=self.network_configuration().network_identifier)
+                Helpers.print_request_body(request, "/key/vote")
+                response: UpdateVoteResponse = api.key_vote_post(request)
+                return self.handle_response(response, print_response)
+            except ApiException as e:
+                Helpers.handleApiException(e)
+    
+    def withdraw_vote(self, print_response=False):
+        with core_api.ApiClient(self.system_config) as api_client:
+            api_client = self.set_basic_auth(api_client, "superadmin", "superadmin")
+            try:
+                api = key_api.KeyApi(api_client)
+                request = UpdateVoteRequest(network_identifier=self.network_configuration().network_identifier)
+                Helpers.print_request_body(request, "/key/withdraw_vote")
+                response: UpdateVoteResponse = api.key_withdraw_vote_post(request)
+                return self.handle_response(response, print_response)
```

Comment:
Ditto - do we want to tell the user that the withdrawal was successful?

Reply:
We generally don't tell users the outcome of their transactions - we just return the tx ids. 

Figuring out whether transactions are successful would require consuming from /transactions, which would complicate the cli further.

Reply:
Gotchya